src/24-hide/pidhide.h

```diff
@@ -5,10 +5,20 @@
 // Simple message structure to get events from eBPF Programs
 // in the kernel to user spcae
 #define TASK_COMM_LEN 16
-struct event {
+#define MAX_PID_LEN 16
+
+// These are used by a number of
+// different programs to sync eBPF Tail Call
+// login between user space and kernel
+#define PROG_00 0
```

Comment:
fix bug for 
![image](https://github.com/eunomia-bpf/bpf-developer-tutorial/assets/19339970/91eb5718-54d7-4c02-972e-d67f041243c3)